cmd/filegen/main.go

```diff
@@ -85,6 +85,15 @@ VERSION:
 		Usage: "Number of initial metachain observers, private/public keys, to generate",
 		Value: 1,
 	}
+	hysteresis = cli.Float64Flag{
+		Name:  "hysteresis",
+		Usage: "Hysteresis value",
```

Comment:
multiplied with numOfNodesPerShard to compute number of nodes allowed in the waiting list of each shard

Reply:
done